wp-content/themes/core/assets/pcss/typography/anchors.pcss

```diff
@@ -0,0 +1,47 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Typography: Anchors
+ *
+ * ----------------------------------------------------------------------------- */
+
+/* -------------------------------------------------------------------------
+ * Anchor: Defaults
+ * ------------------------------------------------------------------------- */
+
+a {
+
+	&:active,
+	&:focus {
+		outline: 1px dotted;
+	}
+}
+
+/* -------------------------------------------------------------------------
+ * Anchor: Inline Links
+ *
+ * Not in theme.json as the selector needs to be more specific
+ * ------------------------------------------------------------------------- */
+
+:where(p, li) a {
```

Comment:
`theme.json` targets very generically when setting default element styles. In addition, the last few projects needed too many :not() selectors to override the fancy inline link styles, I think it's best to target this way.

See [example](https://github.com/wpcomvip/qchi-lendnation/blob/9fce7ba195bd29e5779adf8311c11c471f152213/themes/core/assets/pcss/global/links.pcss#L1) from LendNation